torchvision/csrc/io/image/cpu/decode_jpeg.cpp

```diff
@@ -152,8 +152,23 @@ torch::Tensor decode_jpeg(const torch::Tensor& data, ImageReadMode mode) {
   jpeg_destroy_decompress(&cinfo);
   return tensor.permute({2, 0, 1});
 }
+#endif // #if !JPEG_FOUND
 
+int64_t _jpeg_version() {
+#ifdef JPEG_FOUND
```

Comment:
Is this set even if we build against turbo?

Reply:
yes turbo is fully compatible with the normal jpeg lib so it also defines the same stuff